### PR TITLE
add configuration

### PR DESCRIPTION
**Pull request by SveTTTum**

### CODE REVIEW

File: package-lock.json
Comment (KirylYanovich) [inline package-lock.json:1]: It makes sence to add package-lock.json to .gitignore, as this file is created on project initialization

File: tests/example.spec.js
Comment (KirylYanovich) [inline tests/example.spec.js:1]: The name of file does not represent the content

File: package-lock.json
Comment (SveTTTum) [reply package-lock.json:1]: I think package-lock.json must be in the repository for all team members to work with the same set of dependencies

File: tests/example.spec.js
Comment (SveTTTum) [reply tests/example.spec.js:1]: renamed

**Pull request merged**